arches/app/media/js/views/base-manager.js

```diff
@@ -64,6 +64,20 @@ define([
                 return window.location.pathname === "/search" || window.location.pathname === "/plugins/c8261a41-a409-4e45-b049-c925c28a57da";
             });
 
+            // Register binding of onEnterkeyClick. e.g. <div data-bind="onEnterkeyClick"> </div>
```

Comment:
@phudson-he - Is this part of the solution for the visibility issue solution?

Reply:
Re: @aj-he's comment, I'll approve this then

Reply:
No it isnt